src/PuliPlugin.php

```diff
@@ -466,10 +466,10 @@ private function loadPuliPackages()
         $packages = array();
 
         $output = $this->puliRunner->run(
-            'package --list --format \'%name%;%installer%;%install_path%;%state%\''
+            'package --list --format "%name%;%installer%;%install_path%;%state%"'
```

Comment:
The output of `composer update` with single quotes:
```
Looking for updated Puli packages

  [InvalidArgumentException]
  The package state must be one of "enabled", "not-found", "not-loadable". Got: "enabled'"

install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-plugins] [--no-custom-installers] [--no-a
utoloader] [--no-scripts] [--no-progress] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [--ignore-platform-reqs] [pac
kages1] ... [packagesN]
```

I will take a look if I can run some other command with PuliRunner.